cinn/ir/schedule_desc_test.cc

```diff
@@ -737,5 +737,22 @@ TEST_F(TestScheduleDesc, StepKind_SamplePerfectTile) {
   CheckReplayResult(ir_sch, ir_sch.GetTraceDesc());
 }
 
+TEST_F(TestScheduleDesc, StepKind_SampleCategorical) {
+  lowered_funcs         = LowerCompute({32, 128}, target);
+  ir::IRSchedule ir_sch = MakeIRSchedule(lowered_funcs);
+  Expr ret              = ir_sch.SampleCategorical({1, 2, 3}, {1.0, 2.0, 3.0});
+  std::vector<int> decision;
+  trace.Append(ScheduleDesc::Step("SampleCategorical",
```

Comment:
```suggestion
  std::vector<int> decision = {ret.as_int32()};
  trace.Append(ScheduleDesc::Step("SampleCategorical",
```
此处trace.append需与IRSchedule中append的参数一致，传入的decision为此次sample结果